### PR TITLE
Move zlib to cc image

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -78,9 +78,9 @@ dpkg_list(
         "libgcc1",
         "libgomp1",
         "libstdc++6",
+        "zlib1g",
```
Comment (chanseokoh) [inline WORKSPACE]: There's another instance of `"zlib1g"` under `#dotnet`. Remove that too.

```
        #dotnet
        "libcurl3",
        "libgssapi-krb5-2",
        "libicu57",
        "liblttng-ust0",
        "libssl1.0.2",
        "libunwind8",
        "libuuid1",
        "zlib1g",
        "curl",
```

Comment (loosebazooka) [reply WORKSPACE]: done, but left the one in package_manager/README.md, I'm going to take another look at that file

Comment (chanseokoh) [reply WORKSPACE]: Yeah, I think we can leave the one in `package_manager/README.md`.

**Comment by loosebazooka** [comment]
I'm going to close this for now. I'm less enamored with this solution than I would like.

**Pull request closed without merging**